lib/widgets/pages/home_page.dart

```diff
@@ -34,7 +34,12 @@ class _HomePageState extends State<HomePage> {
     });
 
     try {
-      config = await _configRepository.getAppConfig();
+      final either = await _configRepository.getAppConfig();
+      if (either.success) {
+        config = either.right;
+      } else {
+        config = null;
+      }
     } catch (e) {
       config = null;
```

Comment:
Do we still need the try catch? Can we avoid to have a null config?

Reply:
This code snippet will be removed in another PR which is why i did not bother to fix it :)